Assets/Scripts/UI/PauseScrn/Navigation_Notebook.cs

```diff
@@ -25,11 +25,6 @@ void Start()
 		note2.SetActive (false);
 		note3.SetActive (false);
 		note4.SetActive (false);
-
-		if (note0.activeSelf == false && note1.activeSelf == false && note2.activeSelf == false && note3.activeSelf == false && note4.activeSelf == false) {
```

Comment:
But changing the color causes it to permanently be red.
I added a line in Navigation where we Select the button so that the back gets highlited. Sometimes it doesn't work though :/ but atleast the button isn't always red.